libs/services/auth/testing/src/fixtures/fixture-factory.ts

```diff
@@ -397,6 +430,22 @@ export class FixtureFactory {
     })
   }
 
+  async createPersonalRepresentativeDelegationType({
+    delegationTypeCode,
+    personalRepresentativeId,
+    id,
+  }: {
+    delegationTypeCode: string
+    personalRepresentativeId: string
+    id?: string | null
+  }) {
+    return await this.get(PersonalRepresentativeDelegationTypeModel).create({
```

Comment:
returning with await is redundant unless inside a try/catch. Then you need to await the result to catch if it would throw.

```suggestion
    return this.get(PersonalRepresentativeDelegationTypeModel).create({
```